src/components/MUITypography/Typography.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import { createMuiTheme, ThemeProvider, Typography } from '@material-ui/core';
+import { select, text } from '@storybook/addon-knobs';
+import React from 'react';
+import { ProgressButton } from '../ProgressButton';
+
+export default { title: 'Components/Typography' };
+
+const theme = createMuiTheme({
+  typography: {
+    fontFamily: 'Arial',
+    button: {
+      fontSize: 10,
+    },
+  },
+});
+
+export const Story = () => (
+  <ThemeProvider theme={theme}>
+    <Typography
+      variant={select(
+        'Variant',
+        ['body1', 'caption', 'h1', 'h5', 'subtitle1'],
+        'body1'
+      )}
```

Comment:
It might be more useful to see all the typography variants next to each other so we can visually compare easily.
We should at least include h1-h4, body1/2, caption.